libs/core/shared/types/heading-level.ts

```diff
@@ -0,0 +1 @@
+export type HeadingLevel = 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6';
```

Comment:
good that you worked on this so I noticed a bug :)
with this input we end up with `aria-level="h2"`, which is wrong. aria-level is a number:
`aria-level="2"`